vocabularies/gsq-dataset-themes.ttl

```diff
@@ -0,0 +1,100 @@
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Operational data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:physical-samples a skos:Concept ;
+    skos:definition "Data and metadata pertaining to physical samples taken during geoscience data acquisition e.g. dril core, rock chips, rock samples from outcrop, soil, sediment, water, drilling fluids, oil, gas etc."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Physical samples"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:publications a skos:Concept ;
+    skos:definition "Publications that describe the geology, resource potential, or other areas of departmental interest such as books, journals, periodicals, presentations, posters, academic publications, or multimedia products."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Publications"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:reports a skos:Concept ;
+    skos:definition "Statutory reports lodged for compliance with tenure grant and retention. These may be tenure based such as a resource authority annual report or activity based such as a well completion report."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Reports"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:resource-statistics a skos:Concept ;
+    skos:definition "Statistical data and reports submitted to the department or collected/generated by the department, particulary production and sales quantities."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Resource statistics"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:spatial-data a skos:Concept ;
+    skos:definition "Data and metadata related to spatial objects and representations such as maps, aerial and satellite imagery, spatial data files and databases."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Spatial data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:activities-and-results a skos:Collection ;
+    skos:definition "Primary dataset that describe the geological features of interest and the activities that generate those datasets"@en ;
+    skos:member :geoscience-data,
+        :geoscience-project-data,
+        :operational-data,
+        :physical-samples ;
+    skos:prefLabel "Activities and Results"@en .
+
+:ancillary-information a skos:Collection ;
+    skos:definition "Data descriptive of the organisations involved in data collection and the administrative constraints and associations under which data is collected or related."@en ;
+    skos:member :administrative-data,
+        :spatial-data ;
+    skos:prefLabel "Ancillary Information"@en .
+
+:products-and-documents a skos:Collection ;
+    skos:definition "The final products and official reports that synthesis the collected data into documentation that describe activities and/or provide interpretive and contextual information."@en ;

```

Comment:
In the definition, should the word be "synthesise" not "synthesis"

Reply:
fixed